README.adoc

```diff
@@ -0,0 +1,453 @@
+
+[source,shell]
+----
+RUN pip install dominodatalab
+----
+
+To install a specific version of the library from PyPI, such as `1.0.6`, use the following command:
+
+[source,shell]
+----
+pip install dominodatalab==1.0.6
+----
+
+To install a specific version of the library from GitHub, such as `1.0.6`, use the following command:
+
+[source,shell]
+----
+pip install https://github.com/dominodatalab/python-domino/archive/1.0.6.zip
+----
+
+== Set up the connection
+
+You can set up the connection by creating a new instance of `Domino`:
+
+._class_ Domino(project, api_key=None, host=None, domino_token_file=None, auth_token=None)
+
+.Parameters
+
+* _project:_ A project identifier (in the form of ownerusername/projectname)
+* _api_key:_ (Optional) An API key to authenticate with. 
+If not provided, the library expects to find one in the `DOMINO_USER_API_KEY` environment variable.
+* _host:_ (Optional) A host URL. 
+If not provided, the library expects to find one in the `DOMINO_API_HOST` environment variable.
+* _domino_token_file:_ (Optional) Path to domino token file containing auth token. 
+If not provided, the library expects to find one in the `DOMINO_TOKEN_FILE` environment variable.
+* _auth_token:_ (Optional) Authentication token.
+
+[NOTE]
+====
+* The authentication preference should always be given to the authentication token. 
+If it's not passed, the path to domino token file takes precedence, otherwise the API key is used. 
+If none of these three parameters are passed, then preference is given to the Domino token file from the corresponding environment variable, then to the API key from the corresponding environment variable.
+* By default the log level is set to `INFO`.
+To set the log level to `DEBUG`, set the `DOMINO_LOG_LEVEL` environment variable to `DEBUG`.
+* For testing purposes and issues with SSL certificates, set the environment variable `DOMINO_VERIFY_CERTIFICATE` to `false`. 
+Be sure to unset this variable when not in use.
+====
+
+== Methods
+
+=== Projects
+
+See https://github.com/dominodatalab/python-domino/blob/master/examples/example_projects_usage.py[`example_projects_usage.py`] for example code.
+
+==== project_create(project_name, owner_username=None)
+
+Create a new project with given project name.
+
+.Parameters
+
+* _project_name:_ The name of the project.
+* _owner_username:_ (Optional) The owner username for the project. 
+This parameter is useful when you need to create a project under an organization.
+
+==== collaborators_get()
+
+Get the list of collaborators on a project.
+
+==== collaborators_add()
+
+Add collaborators to a project.
+
+[source,python]
+----
+domino.collaborators_add("collaborator1", "collaborator2")
+----
+
+.Parameters
+
+* _collaborator_username:_ Usernames of one or more collaborators to add to the project.
```

Comment:
@ddl-olsonJD I noticed these in the example code file and added them here.  Let me know if that's not correct.

Reply:
I also tried a different presentation, here and below, to make the headings shorter while still including example syntax below each heading.  Let me know whether that works or whether it's misleading to our audience.  If it works, I'll do the same with the rest of the headings, and if not, I'll change this to match how the existing headings are formatted.